tools/mlir_bench/ov_model_gen.py

```diff
@@ -169,22 +174,28 @@ def generate_ov_model(layers_desc: str, data_type: str, file_name: str,
 
 
 class BaselineMLP(nn.Module):
-    def __init__(self, sizes_mnk, type=None):
+    def __init__(self, sizes_mnk, type=None, layers=3):
         super(BaselineMLP, self).__init__()
         m = sizes_mnk[0]
         n = sizes_mnk[1]
         self.bias = torch.empty((m, n), dtype=type).data.fill_(0.01)
         self.relu = nn.ReLU()
+        self.layers = layers
     def forward(self, a, b):
-        c = torch.matmul(a, b)
-        c = torch.add(c, self.bias)
-        return self.relu(c)
+        for _ in range(0,self.layers):
+            c = torch.matmul(a, b)
+            c = torch.add(c, self.bias)
```

Comment:
I think we should create separate bias per layer.
Otherwise, we'll have fewers args (and packs) and it's not really comparable with standard `nn.Linear`.

Reply:
Done

Reply:
After some testing, it looks like unnecessary complication for now.
They change has been reverted.